CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Added 🚀
 
+-   A new option in the Global Settings allows to enable/disable experimental features #1318
+    -   Click on the settings button in the upper right corner to open the Global Settings dialog.
+    -   Activate/Deactivate the new option "Enable Experimental Features"
+    -   The features will be shown/hidden accordingly
```

Comment:
A general question if old things don't work / are not existent do we want to keep flagging them as experimental?
aka. for me where do we draw the line? 
In my opinion if something is tested and an improvment over the orginal shouldn't we deploy it as a new feature?